### PR TITLE
Details page

### PR DESCRIPTION
**Pull request by Achilles-Dev**
## Changes in this PR:
- [ ] Created the Details page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +21,15 @@ function App() {
...
         />
+        <Route
+          exact="true"
+          path="/motorcycles/:id"
+          element={
+          currentUser.token && currentUser.token !== null
+            ? <DetailsPage userId={currentUser.currentUser.id} />
+            : 'Redirect to Login Page'
+          }
+        />
```
Comment (AyoubM490) [inline src/App.js:39]: Good job adding the authentication check

File: src/components/details/SideView.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const SideView = (props) => {
+  const { motorcycle } = props;
+
+  const deposit = motorcycle.price * 0.05;
+
+  const serviceFee = motorcycle.price * 0.01;
+
+  const financeFee = motorcycle.price * 0.03;
+
```
Comment (AyoubM490) [inline src/components/details/SideView.js:15]: Good job calculating deposit, service fee and finance fee in the script without adding them in the db

**Review by AyoubM490 — APPROVED** [review]
Your project is complete, it's time to merge!

**Review by sentayhu19 — APPROVED** [review]
Hi @Achilles-Dev 
# Status: Approved ✔️ ✔️ ✔️ 
## TO HIGHLIGHT
✔️ Wonderful job on following the design of the details page 🏅 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_Check for optional comments below_

Cheers and Happy coding!👏👏👏

**Pull request merged**